### PR TITLE
Fixing Bug for SecretsManager for Kubecost pattern

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/patterns/secureingresscognito.md [hunk 1]
```diff
@@ -42,11 +42,10 @@ For GitOps, the blueprint bootstrap the ArgoCD addon and points to the [EKS Blue
 1. argo-admin-password secret must be defined as plain text (not key/value) in `us-west-2`  region.
 
 ```
-aws secretsmanager create-secret --name argo-admin-password \
+aws secretsmanager create-secret --name argo-admin-secret \
     --description "Admin Password for ArgoCD" \
-    --secret-string {"$zRrGdeYOCQJ"}
-    --region us-west-2
-    --kms-key-id "aws/secretsmanager"
+    --secret-string "password123$" \
```
Comment (shapirov103) [inline docs/patterns/secureingresscognito.md:47]: i hope it is not the actual password that you used

**Review by shapirov103 — COMMENTED** [review]
one comment

File: docs/patterns/secureingresscognito.md (hunk 1 above)
Comment (elamaran11) [reply docs/patterns/secureingresscognito.md:47]: No it is not.

**Review by shapirov103 — APPROVED** [review]
lgtm

**Pull request merged**